src/models/document/drag-tiles.ts

```diff
@@ -61,6 +61,7 @@ export const DocumentContentModelWithTileDragging = BaseDocumentContentModel
       getTileContentInfo(clonedTile.content.type)?.contentSnapshotPostProcessor?.(clonedTile.content, idMap);
       dragTileItems.push({
         rowIndex, rowHeight, tileIndex,
+        newTileId: idMap[srcTile.id],
```

Comment:
Related to @kswenson's comment, I don't think we want to set the newTileId here. @kswenson pointed out before that a user (in the future) might paste the same tile(s) multiple times. So we really want to generate the newTileId "on paste" or "on drop" instead of "on copy " or "on drag".

It has been weird for a while now, that `cloneTileSnapshotWithNewId` was called "on drag" and then "on drop" a second new id was generated for the tile. So what you've done here is fix that weirdness, but it will cause problem when we support tile copy and paste.

You could try making a new version of `cloneTileSnapshotWithNewId` that doesn't add an id.  Despite its name that function does more than just add a new id. So perhaps we need to keep the other stuff it is doing.

However, if this is working as is, I'm fine with leaving it alone, and then addressing it in the PR that adds shared model drag and drop support.

Reply:
@scytacki @kswenson This was actually a change I made late in the process in response to the jest test that was failing because of incrementing unique ids. I'm not sure if this helped with that problem, but it definitely wasn't the only thing contributing to it. Plus, it seems to be a problem more with the test than the code. I thought about removing it, but figured it was in so might as well leave it at this point. But I'll take it out.